app/workers/data_request_worker.rb

```diff
@@ -58,6 +62,9 @@ def perform(request_id)
 
       request.stored_export.upload(path)
       request.complete!
+    rescue DataRequest::DataRequestCanceled
+      binding.pry
```

Comment:
👻

Reply:
i can't believe i left a binding.pry in there!